### DISCUSSION TITLE
CSV Data

### DISCUSSION THREAD

**Discussion by revsean** [AutomaApp/automa#230] [Q&A]
### Get Data from Downloaded CSV into Automa  

I am trying to automate using Automa (great plugin). I had it downloading a CSV file for me, but I am wondering if I can get it just directly ingest the data and then be able to spit it out for me in a new place. I am not a coder so I am learning on the fly.

**Comment by Kholid060** [comment]
Automa can't get data from a downloaded file, so you have to do it manually
